app/main.py

```diff
@@ -1,3 +1,45 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: (int, float)) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: (int, float)) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        return Distance(self.km + other)
+
+    def __iadd__(self, other: (int, float, Distance)) -> Distance:
+        if isinstance(other, Distance):
+            self.km += other.km
+        else:
+            self.km += other
+        return self
+
+    def __mul__(self, other: (int, float)) -> Distance:
+        return Distance(self.km * other)
+
+    def __truediv__(self, other: (int, float)) -> Distance:
+        return Distance(round(self.km / other, 2))
+
+    def __lt__(self, other: (int, float)) -> bool:
+        return self.km < other
+
+    def __gt__(self, other: (int, float)) -> bool:
+        return self.km > other
```

Comment:
What if `other` is of type `Distance` here and in all the following methods?

Reply:
In my first solution I check it, but @YuriiKindrat  told me that other can not be of type Distance in this method, because if you multiply km * km you get square km.
https://github.com/mate-academy/py-distance-class/pull/359#discussion_r1097762848
According our condition distance < Distance(60), so other can be just class int.